models/isam/interfaces/isam_interfaces.yml

```diff
@@ -0,0 +1,70 @@
+module: isam_interfaces
+version_added: 2.9
+short_description: 'Manages interface attributes of Nokia ISAM MSAN devices.'
+description: 'This module manages interface attributes of Nokia ISAM MSAN devices'
+author: Jan Kuehnemund
+notes:
+- 'Tested against Nokia ISAM with OS Version R6.2.04m'
+options:
+  config:
+    description: A dictionary of options for interface ports
+    type: list
+    elements: dict
+    suboptions:
+      id:
+        type: str
+        description: 
+        - configure a specific interface port
+      admin-up:
```

Comment:
@jahknem Module options are recommended to be valid python identifiers. Could you change the hyphens to underscores wherever applicable?

Reply:
I think I found all applicable hyphens